root/layout/components/Search.js

```diff
@@ -44,7 +44,7 @@ const Search = () => (
     {' '}{searchOptions}{' '}
     <input type="hidden" id="headerid-method" name="method" value="indexed" />
     <button type="submit">
-      <img src={manifest.pathTo('/images/icons/search.svg')} />
+      <img alt={l('Search')} src={manifest.pathTo('/images/icons/search.svg')} />
```

Comment:
I think I would use an empty `alt=""` here (and in the TT below, of course). We already have “Search“ as the placeholder text for the input field, so it would be unnecessary clutter for a screenreader to repeat it here. That it is the submit button for the search form should be enough.